packages/api-explorer/src/JsonForm/configureJsonEditor.js

```diff
@@ -34,14 +38,16 @@ function setDefaultCustomization (JSONEditor) {
         }
 
         if (this.container !== undefined && this.title !== undefined) {
-          const classContainer = (key === 'upload' || key === 'base64') ? 'mia-container-upload-file-wrapper' : 'mia-container-wrapper' 
+          const classContainer = (key === 'upload' || key === 'base64') ? 'mia-container-upload-file-wrapper' : 'mia-container-wrapper'
           this.container.classList.add(classContainer)
         }
       }
     }
   });
 }
 module.exports = function configureJSONEditor(JSONEditor, intl, setFormSubmissionListener) {
+  if (JSONEditor.hasBeenEdited) return
```

Comment:
It's passed here only after the initial configuration and the for the edit of something?

Reply:
https://github.com/mia-platform/api-explorer/pull/78/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.snap#r445461406

Reply:
We invoke `configureJSONEditor` on JsonForm mount. JsonForm is instanced for each "Form" (Body, Query fields, params, ecc).
This method changes some JSONEditor global options (example: JSONEditor.editors).
This means that we made globals changes for each form, but we really need to do it once.
This multiple changes caused (we don't get it why) the bug linked by Daniele

Reply:
What happen if there is the same field with different type in more forms?

Reply:
Each Form has its own instance of JsonForm. 
Only the configurations (themes, editors, options) are globals

Reply:
It is a strange behaviour to see the number of type equal of the number of form rendered. But it's better to continue the conversation in the thread below